### PR TITLE
feat(timepicker): add styling for timepicker

### PR DESCRIPTION
**Pull request by inikolova**

### CODE REVIEW

File: scss/datetime/_layout.scss [hunk 1]
```diff
@@ -1,4 +1,25 @@
+// Infinite calendar
+@mixin hide-scrollbar($dir: 'right') {
```
Comment (gyoshev) [inline scss/datetime/_layout.scss]: We have the same code in `calendar/_layout.scss`. Extract it to `mixins/core/` and reuse it, to reduce duplication. If the different `$max-scrollbar` value is needed, use a mixin parameter.

File: scss/datetime/_layout.scss [hunk 2]
```diff
@@ -118,4 +139,95 @@
 
     }
 
+    // Infinite timepicker
+    // Header
+    .k-time-header { // Same as the calendar header
```
Comment (gyoshev) [inline scss/datetime/_layout.scss]: If the header is the same by design, perhaps we can use the `k-calendar-header` class? There are also duplicated styles in the _theme file, with k-time-now acting as k-today.

**Review by gyoshev — COMMENTED** [review]
It seems that some of the calendar styling can be reused, which will trim down the styles and reduce duplication

File: scss/datetime/_layout.scss (hunk 2 above)
Comment (inikolova) [reply scss/datetime/_layout.scss]: After further discussion with @ggkrustev , we decided to use `<button>` for the Now action in the timepicker, which differs from the Calendar component (where `<span>` is used). I will modify the styling in the timepicker header

Comment (gyoshev) [reply scss/datetime/_layout.scss]: Does the same reasoning apply to the calendar component? Using a `<button>` might be better for the calendar accessibility, too.

**Pull request merged**